### DISCUSSION TITLE
Is there a way to sideload extension from storage account

### DISCUSSION THREAD

**Discussion by qinl-li** [microsoft/vscode-discussions#814] [Extension Development QnA] [answered]
We have an internal extension and want to side-load in vscode.dev. It seems like vscode.dev has restrictions on the domain. 

![image](https://github.com/microsoft/vscode-discussions/assets/63257243/91e66639-8503-4841-b448-cdae8b17996e)

Is there a workable solution?

**Comment by alefragnani** [comment, marked as answer]
Hi @qinl-li ,

There was a similar discussion reported a few months ago, but with the intention to test extensions, and, as answered in this comment https://github.com/microsoft/vscode-discussions/discussions/596#discussioncomment-6356367, it seems yes, there are domain restrictions.

The same comment provides the updated steps to test extension on web (https://code.visualstudio.com/api/extension-guides/web-extensions#test-your-web-extension). So, you could follow these steps to install your internal extension (locally), but yes, these doesn't seems to be a viable solution for _publishing_ the internal extension for your internal colleagues.

Based on this domain restrictions, I'm not sure vscode.dev is intended to have the same openess for sideloading capabilities as the regular Desktop app. I guess you won't be able to use your internal extension on it.

Hope this helps
